arch/risc-v/src/common/riscv_addrenv.c

```diff
@@ -81,6 +81,12 @@
 #  error "This module is intended to be used with CONFIG_BUILD_KERNEL"
 #endif
 
+/* Make sure the address environment virtual address boundary is valid */
```

Comment:
Please move "#define ADDRENV_VBASE       (CONFIG_ARCH_DATA_VBASE)" to above line 84. The CI is breaking!